src/main/kotlin/Shovel.kt

```diff
@@ -15,6 +15,19 @@ fun shovelAsync(kafka: KafkaConsumer<ByteArray, ByteArray>, hbase: HbaseClient,
             kafka.subscribe(Config.Kafka.topicRegex)
             val records = kafka.poll(pollTimeout)
             for (record in records) {
+
+                val newKey: ByteArray = record.key() ?: ByteArray(0)
+                if (newKey.isEmpty()) {
+                    log.warning(
+                        "Empty key '%s' was skipped for %s:%d:%d".format(
```

Comment:
is there really any point logging the key here? :D

Reply:
Changed. For note, I said this too but Mark was insistent :D